api/v1beta1/kafkacluster_types.go

```diff
@@ -138,6 +138,15 @@ type RollingUpgradeConfig struct {
 	// distinct broker replicas with either offline replicas or out of sync replicas and the number of alerts triggered by
 	// alerts with 'rollingupgrade'
 	FailureThreshold int `json:"failureThreshold"`
+
+	// ConcurrentBrokerRestartsAllowed controls how many brokers can be restarted in parallel during a rolling upgrade. If
+	// it is set to a value greater than 1, the operator will restart up to that amount of brokers in parallel, if the
+	// brokers are within the same AZ (as specified by "broker.rack" in broker read-only configs). Since using Kafka broker
```

Comment:
```suggestion
	// brokers are within the same rack (as specified by "broker.rack" in broker read-only configs). Since using Kafka broker
```

Reply:
Addressed in 7ed1df6ef4904c0e909da364d701b9031d014a9e